pkg/apis/app/v1beta1/application_types.go

```diff
@@ -84,6 +84,9 @@ type ApplicationSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
 	Selector *metav1.LabelSelector `json:"selector,omitempty"`
 
+	// AddOwnerRef objects - flag to indicate if we need to add OwnerRefs to matching objects
+	AddOwnerRef bool `json:"addOwnerRef,omitempty"`
```

Comment:
Could you add this field in the documentation / readme too ?

Reply:
Done. Added.